src/components/appBar/CyVerseAppBar.js

```diff
@@ -225,7 +202,10 @@ function SearchOptions(props) {
                 <Button
                     id={build(ids.APP_BAR_BASE_ID, ids.SEARCH_FILTER_BTN)}
                     onClick={handleClick}
-                    style={{ backgroundColor: theme.palette.white }}
+                    style={{
+                        backgroundColor: theme.palette.primary.main,
+                        color: theme.palette.white,
```

Comment:
> LGTM 👍
> 
> I made a few minor `theme` suggestions, and it seems like we need i18n messages in the app bar.
> I also agree with @ash3rz that we should try out a relative height for the table contents.

Thanks for reviews. I am gonna close this PR for now since I still have to get it working with small screens and do some cleans ups.